lib/Parser/parse.cpp

```diff
@@ -7708,7 +7708,7 @@ void PidRefStack::TrackAssignment(charcount_t ichMin, charcount_t ichLim)
         {
             return;
         }
-        Assert(this->GetIchLim() >= ichLim);
+        Assert(ichMin <= this->GetIchMin() && this->GetIchLim() <= ichLim)
```

Comment:
Yeah, I have done that do..while change locally - will do different PR for that.